lib/CodeGen/CGOpenMPRuntime.cpp

```diff
@@ -905,7 +905,7 @@ Address CGOpenMPRuntime::getOrCreateDefaultLocation(unsigned Flags) {
       DefaultOpenMPPSource =
           CGM.GetAddrOfConstantCString(";unknown;unknown;0;0;;").getPointer();
       DefaultOpenMPPSource =
-          llvm::ConstantExpr::getBitCast(DefaultOpenMPPSource, CGM.Int8PtrTy);
+          llvm::ConstantExpr::getPointerCast(DefaultOpenMPPSource, CGM.Int8PtrTy);
```

Comment:
If OpenMP would like to adopt this branch, it will need this change to work since in this branch CGM.Int8PtrTy is a generic pointer.